datafusion/physical-expr/src/expressions/datetime.rs

```diff
@@ -77,7 +77,7 @@ impl PhysicalExpr for DateTimeIntervalExpr {
     }
 
     fn data_type(&self, input_schema: &Schema) -> Result<DataType> {
```

Comment:
I would also love to remove `DateTimeIntervalExpr` out of its own thing and into binary.rs with the other operators, but for now 👍